src/__tests__/backend/space-test.js

```diff
@@ -127,6 +128,82 @@ describe('App Spaces', () => {
       expect(props.created_at).toBeDefined();
       expect(props._parent).not.toBeDefined();
     });
+
+    // bot stuff 
+
+    it('should not have any bots', () => {
```

Comment:
I have been breaking out tests in `diag-app` as a file per data type, that way they can operate independently. This has some downsides (tests take longer to run etc). Probably needs a complete rethink, but this does break the pattern.

Reply:
yes it does :)